package.json

```diff
@@ -48,9 +48,9 @@
 	"devDependencies": {
 		"@axe-core/react": "^4.3.1",
 		"@datadog/datadog-api-client": "^1.0.0-beta.7",
-		"@hashicorp/platform-cli": "^2.4.0-canary-2022816231816",
+		"@hashicorp/platform-cli": "^2.4.0-canary-2022101823752",
 		"@hashicorp/platform-configs": "^0.2.0",
-		"@hashicorp/platform-nextjs-plugin": "^3.1.0",
+		"@hashicorp/platform-nextjs-plugin": "^4.0.0-canary-20221023213744",
```

Comment:
Are these going to be stable releases after we upgrade?

Reply:
Yeah, just gotta merge the versioning PR to publish them 👍